### DISCUSSION TITLE
Integration in React Native

### DISCUSSION THREAD

**Discussion by neeraj-tangariya** [jeffreylanters/react-unity-webgl#492] [Question and Answer] [answered]
Hi @jeffreylanters, I'm trying to integrate this package in my react-native application but it throws error?
Does this package work in React Native?
please let me know...

**Comment by jeffreylanters** [comment, marked as answer]
Hi. No, sorry. This package is only compatible with WebGL environments such as the browser.

**Reply by neeraj-tangariya** [reply]
Thank @jeffreylanters for you response, 
Is there any alternative that I can integrate unity with React-Native????

could you please suggest any other package. I found some package but that are not maintained.

**Reply by jeffreylanters** [reply]
I've never really looked into it, I know there are some packages that did it in the past, but as I remember they required quite some work to just get the game running due to Unity's native build complexity.

So I don't think I can really help you out on this one, sorry!
